doc/user-manual/tools/command-line-options.rst

```diff
@@ -1491,6 +1491,66 @@ The list containing any warning ``NAME`` can be produced by ``agda --help=warnin
 
      :ref:`NO_TERMINATION_CHECK<terminating-pragma>` pragmas; such are deprecated.
 
+.. option:: RewriteLHSNotDefOrConstr
+
+     Rewrite rule head symbol is not a defined symbol or constructor.
```

Comment:
From the explanation, this looks like a duplicate of `RewriteHeadSymbolNotPostulateFunctionConstructor`.
Also, maybe `Def` and `Constr` should be expanded (unabbreviated) for sake of uniformity.

Reply:
Actually the other error (`RewriteHeadSymbolNotPostulateFunctionConstructor`) is used only when the head symbol of a rewrite rule is a type constructor, so I'll rename that one to `RewriteHeadSymbolIsTypeConstructor` and expand the Def and Constr in the name of this one.